surf/apps/communities/models.py

```diff
@@ -49,14 +66,20 @@ class Community(UUIDModel):
     Implementation of Community model. Communities are related to
     SURFconext Teams.
     """
-
-    surf_team = django_models.OneToOneField(SurfTeam,
-                                            on_delete=django_models.CASCADE,
-                                            related_name="community",
-                                            null=True)
-
+    publish_status = enum.EnumField(PublishStatus, default=PublishStatus.DRAFT)
     name = django_models.CharField(max_length=255, blank=True)
     description = django_models.TextField(blank=True)
+    title_translations = django_models.OneToOneField(to=Locale, on_delete=django_models.CASCADE,
+                                                     null=True, blank=False)
```

Comment:
Add blank=True here, because for now people will mostly use one language. Having to create fake object to satisfy admin's greedy needs is a bit over the top.

Reply:
Done, however I noticed later on that it's logical to also do this for the description_translation so that's another migration.